### DISCUSSION TITLE
Not working

### DISCUSSION THREAD

**Discussion by wonderfulForest** [mircokroon/minecraft-world-downloader#408] [Q&A]
I'm on Windows 10 and I downloaded Java JDK from the link in the description (of this video: https://www.youtube.com/watch?v=Hjz2tN-AWOE), but when I try to run the world-downloader.exe, nothing happens at all (at least nothing that I can see). I've tried restarting my computer, reinstalling world-downloader.exe, and even running world-downloader.exe as an administrator but nothing is working. Help please?

**Comment by mircokroon** [comment]
You could try running the .jar from the commandline, I'm not sure what the issue might be.

**Reply by mctogs** [reply]
I am having the same issue and when I tried to run from cmd it gave me this error: 

Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.UnsupportedClassVersionError: Launcher has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 52.0

I am running the latest version of Java, is the mod not up to date with the latest version?

**Reply by mircokroon** [reply]
Probably you've got multiple versions of Java and it's using an older one, I'd suggest removing all older versions if you don't need them.
